### PR TITLE
Rework listening post access

### PR DESCRIPTION
**Pull request by Tarmunora**
## About the PR 
Agent card no longer grants access to listening post
Traitors can now use the handscanner to open the listening post

Listening post door become open-access once opened for the first time with the hand scanner. (i.e.: if traitors are *actively operating* out of the listening post, it becomes fair game for the crew to launch a counter-raid on it in response)

## Why's this needed? 
Agent card was essentially acting as a tax on entering the listening post, which seemed a bit odd considering sleepers could get in for free

Additionally, the listening post being nigh-impregnable to entry causes friction when traitors use the space to set up teleporter/cloner/lawrack/etc shenanigans with next to no counterplay options from the crew.

## Changelog 

```changelog
(u)tarmunora
(*)Agent card no longer grants access to listening post, traitors and sleeper agents can now use the handscanner to permanently unlock the listening post doors.
```

### CODE REVIEW

**Comment by Cherman0** [comment]
I'm a fan of it, though I worry about pre-emptive checks of the listening post to see if the door is unlockedd. Might be worth clarifying the listening post rules regarding that.

**Comment by Tarmunora** [comment]
> I'm a fan of it, though I worry about pre-emptive checks of the listening post to see if the door is unlockedd. Might be worth clarifying the listening post rules regarding that.

Eh, more or less covered by the same metagaming rules that stop people from checking the door at 5 minutes for fingerprints. But yeah, some clarifications to the page will be made if/when this get fully merged

File: code/obj/machinery/door/door_control.dm [hunk 1]
```diff
@@ -542,7 +552,7 @@ ADMIN_INTERACT_PROCS(/obj/machinery/door_control, proc/toggle)
 	if (ON_COOLDOWN(src, "scan", 2 SECONDS))
 		return
 	playsound(src.loc, 'sound/effects/handscan.ogg', 50, 1)
-	if (user.mind?.get_antagonist(ROLE_SLEEPER_AGENT))
+	if (user.mind?.get_antagonist(ROLE_SLEEPER_AGENT) || user.mind?.get_antagonist(ROLE_TRAITOR))
```
Comment (JORJ949) [inline code/obj/machinery/door/door_control.dm:555]: Spy Thiefs/Nuclear Operatives also need access

Comment (Tarmunora) [reply code/obj/machinery/door/door_control.dm:555]: Hm yup nukies need access yeah.
Spythief can get fuked though 😈

**Comment by JORJ949** [comment]
Id prefer if the access was removed for a moment so you can get in and out (maybe just completely remove the teleporter access requirement) and the door was no longer reinforced/hardened so they still have to break in to raid the post. Especially considering  on classic now if you see "Listening Post Announcement" (said post announcement also still needs an agent card) you know its open and you can go get free syndie gear as the clothing vendor isnt locked (so that sleepers could use it)

**Comment by Glamurio** [comment]
As someone who's seen gimmicks that abused the fact that the listening post was a closed-off space by teleporting unsuspecting victims for free kills, I very much like this. I also like that we no longer need agent cards, I've tried to receive a similar result with a different PR once, but this is just better period.

I'm not worried about pre-emptive raids because that's against the rules, just ahelp it when you see it happen. Lockers are (most of the time) not camped either even though everyone knows that's where the gang base is.

**Comment by TheRyeGuyWhoWillNowDie** [comment]
As a whole, not a fan of this. Traitors being able to get in for free means a 1000% increase of really strong syndicate shotgun pods, it being permanently opened kinda ruins it as a safe spot for syndicates to chill out (Someone else mentioned it being used as a telesci base, not sure how since it doesn't have a mainframe connection) And removing agent ID access means that non-traitor antagonists who got an agent ID elsewhere can't get in anymore either.

**Comment by PastaSpaghett** [comment]
I like the idea of making the listening post more accessible to traitors, but the unlocking of the listening post is far more detrimental to the people using the listening post as intended than those who were using it to harass the station with no repercussions, I don't think this is the best way to handle this issue.

**Comment by JORJ949** [comment]
Can nukies get out of the teleporter room at the moment, iirc theres no scanner in the teleporter room (and i never really got why it was locked in the first place, you already have to get past the first door and the only thing in there is an oxygen tank)

**Comment by Tarmunora** [comment]
> Can nukies get out of the teleporter room at the moment? iirc theres no scanner in the teleporter room (and i never really got why it was locked in the first place, you already have to get past the first door and the only thing in there is an oxygen tank)

yes

**Pull request merged**